content/en/docs/concepts/security/overview.md

```diff
@@ -65,6 +65,8 @@ Google Cloud Platform | https://cloud.google.com/security/ |
 IBM Cloud | https://www.ibm.com/cloud/security |
 Microsoft Azure | https://docs.microsoft.com/en-us/azure/security/azure-security |
 VMWare VSphere | https://www.vmware.com/security/hardening-guides.html |
+Oracle Cloud Infrastructture |  https://docs.cloud.oracle.com/en-us/iaas/Content/Security/Concepts/security.htm |
```

Comment:
Hi. I think you've got a spelling error in “Infrastructture”.

Reply:
Yes that is correct.
It should read "Oracle Cloud Infrastructure"

Reply:
@rishgh would you be willing to fix the typo? Please also strongly consider adding the entry to match the existing sort order.